main.py

```diff
@@ -1,12 +1,216 @@
+            for video in video_store.list_videos():
+                print(video)
+        
+        # Option 5: get information about one video 
+        elif choice=="5":
+            print(option_num(choice, options))
+            video = select_video(video_store)
+            if video == None:
+                print(not_found())
+            else:
+                for attr,info in video.items():
+                    print(f"{attr}: {info}")
+
+        # Option 6: add a customer 
+        elif choice=="6":
+            print(option_num(choice, options))
+            name, postal_code, phone = get_customer_data()
+            if not is_customer_data_valid(name, postal_code, phone):
+                print(not_valid_data())
+            else:
+                customer = video_store.add_customer(name=name, postal_code=postal_code, phone=phone)
+                print(f"Customer id #{customer['id']} has been added.") 
+            
+        # Option 7: edit a customer 
+        elif choice=="7":
+            print(option_num(choice, options))
+            customer = select_customer(video_store)
+            if customer == None:
+                print(not_found())
+            else:
+                name, postal_code, phone = get_customer_data()
+                video_store.update_customer(name=name, postal_code=postal_code, phone=phone)
+                print(f"Updates have been made to customer {name}.")
+
+        # Option 8: delete a customer 
+        elif choice=="8":
+            print(option_num(choice, options))
+            customer = select_customer(video_store)
+            if customer == None:
+                print(not_found())
+            else:
+                video_store.delete_customer()
+                print(f"Customer {customer} has been deleted.")
+
+        # Option 9: get information about one customer 
+        elif choice=="9":
+            print(option_num(choice, options))
+            customer = select_customer(video_store)
+            if customer == None:
+                print(not_found())
+            else:
+                for attr,info in customer.items():
+                    print(f"{attr}: {info}")
+
+        # Option 10: get information about all customers
+        elif choice=="10":
+            print(option_num(choice, options))
+            for customer in video_store.list_customers():
+                print(customer)
+
+        # Option 11: check out a video to a customer
+        elif choice=='11':
+            print(option_num(choice, options))
+            customer = select_customer(video_store)
+            video = select_video(video_store)
+            if customer == None or video == None:
+                print(not_found())
+            else: 
+                rental = video_store.check_out_video(customer_id=customer['id'],video_id=video['id'])
+                print(f"Video #{rental['video_id']} has been checked out to customer #{rental['customer_id']}.")
```

Comment:
This generates a KeyError for `rental['video_id']`.  This is because there is no `video_id` here.